modules/ldapbackend/ldapbackend.cc

```diff
@@ -697,7 +695,7 @@ bool LdapBackend::getDomainInfo( const DNSName& domain, DomainInfo& di )
     // search for SOARecord of domain
     filter = "(&(associatedDomain=" + toLower( m_pldap->escape( domain.toStringRootDot() ) ) + ")(SOARecord=*))";
     m_msgid = m_pldap->search( getArg( "basedn" ), LDAP_SCOPE_SUBTREE, filter, attronly );
-    m_pldap->getSearchEntry( msgid, result );
```

Comment:
Just a nit but it looks like `msgid` is now unused so we should just remove it.

Reply:
@goestreicher Could you please address the nit mentioned above? Then I'll merge it.